pages/search.js

```diff
@@ -134,10 +134,11 @@ const Search = ({ appContext }) => {
           content: query.content || 'all',
           group: query.group || 'all',
           source: Object.keys(sourceOptions).includes(query.source) ? query.source : 'all',
+          sort: 'tmdate:desc',
```

Comment:
i think only of sort here and sort in options is required?
it's taking options.sort or data1.sort

Reply:
Ah you are correct, I can remove it from the options object.